File: dataset_reduzido/codenet/Python/Code/s622795537.py
n = int(input())
arr = {}

for i in range(n):
    s = input()
    if not s in arr:
        arr[s] = 1
    else:
        arr[s] += 1

max_count = max(arr.values())
dic = [key[0] for key in arr.items() if key[1] == max_count]

out_sort = sorted(dic)
for i in out_sort:
    print(i)

File: output/Llama-2-100/codenet/Python/Java/s622795537.java
import java.util.*;

public class s622795537 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        Map<String, Integer> arr = new HashMap<>();

        for (int i = 0; i < n; i++) {
            String s = sc.next();
            if (!arr.containsKey(s)) {
                arr.put(s, 1);
            } else {
                int count = arr.get(s);
                arr.put(s, count + 1);
            }
        }

        int maxCount = Collections.max(Arrays.asList(arr.values()));
        List<String> dic = new ArrayList<>();
        for (Map.Entry<String, Integer> entry : arr.entrySet()) {
            if (entry.getValue() == maxCount) {
                dic.add(entry.getKey());
            }
        }

        Collections.sort(dic);
        for (String s : dic) {
            System.out.print(s + " ");
        }
    }
}
